src/helperFunctions/getAssigneeProfilePic.ts

```diff
@@ -0,0 +1,16 @@
+import { USERS } from '@/components/constants/url';
+import fetch from '@/helperFunctions/fetch';
+import { IMAGE_NOT_FOUND_ERROR_MESSAGE } from '@/components/constants/messages'
+
+const getAssigneeProfilePic = async (username: string) => {
+     try {
+          const url = `${USERS}/${username}`;
+          const { requestPromise } = fetch({ url });
```

Comment:
why awaiting on `requestPromise `and not on `fetch `directly?

Reply:
The **requestPromise** is destructured and awaited instead of the fetch Promise to asynchronously process the response data.